### ISSUE TITLE
All messages are sent at the same time

### ISSUE THREAD

**Issue by s1h4d0w** [montlikadani/AutoMessager#22]
### Problem
All messages in announces.yml are sent at once. I've also tried with a messages.txt file I used with an earlier version but that also sends everything at once. I can't find any issue with my config, I assumed there would be a "send x messages per announce" setting but nothing.

### Details
Plugin version: 2.5

Software version: Paper 199

### Configuration file(s)

config.yml: https://pastebin.com/wDuBLCa8
announces.yml: https://pastebin.com/UW0dGynX (there's an empty line at the end, pastebin just removes it)

### Screenshots (optional)

![image](https://user-images.githubusercontent.com/5654654/82923903-dfa2be80-9f7b-11ea-8321-3155a3e54966.png)

**Comment by montlikadani** [comment]
Fixed here https://github.com/montlikadani/AutoMessager/commit/7308fca37f9c4c71b29c7fc8de50df21bb51906d

**Comment by s1h4d0w** [comment]
As in an update is coming soon? I sadly don't know how to compile a plugin myself.

**Comment by montlikadani** [comment]
Will do that if I can fix some fixes in the plugin.

**Comment by s1h4d0w** [comment]
Awesome thanks! One more thing tho, maybe you already know about it.

As the commit mentions "when there is more players than 2" I set the `min-players` to 0 curious if that would fix it. Normally 0 disables the feature so it should always send messages regardless of player count, but now I've set it to 0 it doesn't send messages at all anymore.

Fine for me for now as I want to disable it until there's a fix, but I assume it's not supposed to work like that.
